Makefile

```diff
@@ -42,6 +42,18 @@ nodelink-controller:
 machine-healthcheck:
 	$(DOCKER_CMD) ./hack/go-build.sh machine-healthcheck
 
+.PHONY: gen-crd
+gen-crd:
+	cp -r vendor/github.com/openshift/cluster-api/pkg/apis/machine pkg/apis/.
```

Comment:
could we may be run the cp, run and rm steps against a tmp folder that we add to our .gitignore list

Reply:
Maybe we should add these commands to a script in hack/ so we don't clutter up the make file.

Reply:
We can't unless we recreate separate GOPATH/src/... workspace. Lemme try it.

Reply:
Done. Copying files under temp dir and generating the CRDs there.